miner/worker.go

```diff
@@ -238,8 +276,11 @@ func (w *worker) close() {
 }
 
 // newWorkLoop is a standalone goroutine to submit new mining work upon received events.
-func (w *worker) newWorkLoop() {
-	var interrupt *int32
+func (w *worker) newWorkLoop(recommitInterval time.Duration) {
+	var (
+		interrupt   *int32
+		minInterval = recommitInterval // minimal resubmit interval specified by user.
```

Comment:
`minInterval` doesn't really say what it's used for, lets rename both to:

`recommitInterval` -> `recommit`
`minInterval` -> `minRecommit`